### ISSUE TITLE
`set-output` is deprecated and will be disabled soon

### ISSUE THREAD

**Issue by LostNeophyte** [check-spelling/check-spelling#37]
`check-spelling` generates this warning:
> The set-output command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

You can see the warning on the bottom [here](https://github.com/LostNeophyte/lunarvim.org/actions/runs/3546310725)

**Comment by jsoref** [comment]
I could create an alpha release tag if you're up for testing it....

**Comment by LostNeophyte** [comment]
Sure, why not, thanks

**Comment by jsoref** [comment]
Here you go: https://github.com/check-spelling/check-spelling/releases/tag/v0.0.21-alpha1.

Note that you'll want to refresh your workflow based on https://github.com/check-spelling/spell-check-this/blob/prerelease/.github/workflows/spelling.yml (use `@v0.0.21-alpha1` for the action, and copy over the `with:` bits you're using) as things have improved in the job handling since v0.0.20.

**Comment by LostNeophyte** [comment]
Other than the errors below, it works fine, thanks for the quick migration guide

There were a couple of these:
```
Error: Failed to retrieve cspell:ada/ada.txt -- https://raw.githubusercontent.com/streetsidesoftware/cspell-dicts/main/dictionaries/ada/ada.txt (dictionary-not-found)
```
But I set `check_extra_dictionaries` to `''` and it's working

**Comment by jsoref** [comment]
Ah, I did intend to include a migration item for it, please try the (updated) instructions in: https://github.com/check-spelling/check-spelling/releases/tag/v0.0.21-alpha1 and see if:
* they work
* they're clear enough

If they aren't, please let me know and I'll work on them.

**Comment by LostNeophyte** [comment]
it works, only this section seemed a bit not clear: 

>If you like a set of cspell: items, you can change your prefixes to cspell0: or anything that isn't cspell: and then set dictionary_source_prefixes to something like:{"cspell": "https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20220816/dictionaries/", "cspell0": "https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20220427/dictionaries/"}.

I'd suggest rewording it to something more like this (if I understood it correctly):


If you want to keep your current cspell `extra_dictionaries`, rename "cspell" to "cspell_old" (or anything that isn't "cspell") and set `dictionary_source_prefixes` to something like this: `{"cspell": "https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20220816/dictionaries/", "cspell_old": "https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20220427/dictionaries/"}`.

**Comment by jsoref** [comment]
Thanks!

I've applied roughly that. Give it a quick read.

I'm currently running through some last minute checks (specifically `act` support is fairly broken, it isn't something I use often, so this isn't surprising, but I try not to leave it broken when I ship....)

**Comment by LostNeophyte** [comment]
you can close the issue if you want, I'd personally keep it open until the full release for other people that might have this problem

Thanks for everything :)

**Comment by jsoref** [comment]
Thanks again for the feedback.

I'm happy to leave issues open.

One thing I try to do as part of the release process is review open issues and close any that are resolved with a pointer to the release.

**Comment by jsoref** [comment]
https://github.com/check-spelling/check-spelling/releases/tag/v0.0.21
https://github.com/check-spelling/check-spelling/commit/2fbb344fb906b4ce3aa1653b8fb6c70e175e1955
